### PR TITLE
Hashed password import in users API: editing for clarity

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:
- **What's changed?** Editing the section on importing hashed passwords to try to make it clearer.
- **Is this PR related to a Monolith release?** No.
### Resolves:

* [OKTA-191976](https://oktainc.atlassian.net/browse/OKTA-191976)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -3645,16 +3645,17 @@ The password specified in the value property must meet the default password poli
 
 ##### Hashed Password Object
 
-Specifies a hashed password that can be imported into Okta.  This allows an existing password to be imported into Okta directly from some other store.
-A hashed password may be specified in a Password Object when creating or updating a user, but not for other operations.  When updating a user with a hashed password the user must have the `STAGED` status.
+Specifies a hashed password to import into Okta. This allows an existing password to be imported into Okta directly from some other store. Okta supports the BCRYPT, SHA-512, SHA-256, SHA-1, and MD5 hashing functions for password import. A hashed password may be specified in a Password Object when creating or updating a user, but not for other operations.  See [Create User with Imported Hashed Password](#create-user-with-imported-hashed-password) for information on using this object when creating a user. When updating a user with a hashed password the user must have the `STAGED` status.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md]: The last sentence I struggled with a bit. There's nothing glaringly wrong with it, I just stumble over it each time I read it. I am suggesting the following, but take it or leave it if you like yours better:

When updating a user with a hash password, their status must be set to `STAGED`.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 2]
```diff
@@ -3645,16 +3645,17 @@ The password specified in the value property must meet the default password poli
...
+| value      | String   | For SHA-512, SHA-256, SHA-1, MD5: This is the actual base64-encoded hashed password. For BCRYPT: This is the actual radix64-encoded hashed password.                                       | TRUE                                                                          | N/A                            | N/A                            |
+| salt       | String   | For SHA-512, SHA-256, SHA-1, MD5: Specifies the base64-encoded password salt used to generate the hash. For BCRYPT: Specifies the radix64-encoded password salt used to generate the hash. | TRUE                                                                          | 22 (only for BCRYPT algorithm) | 22 (only for BCRYPT algorithm) |
+| workFactor | Integer  | Governs the strength of the hash, and the time required to compute it. Only relevant for BCRYPT algorithm.                                                                                 | Only for BCRYPT algorithm                                                     | 1                              | 20                             |
```
[the remark this answers is not in the document]
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: Made the fix.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 3]
```diff
@@ -3645,16 +3645,17 @@ The password specified in the value property must meet the default password poli
...
-| workFactor   | Integer    | Governs the strength of the hash, and the time required to compute it. Only relevant for `BCRYPT` algorithm                                                                               | Only for `BCRYPT` algorithm                                 | 1                                | 20                               |
-| saltOrder    | String     | Specifies whether salt was pre- or postfixed to the password before hashing. Only relevant for `SHA-512`, `SHA-256`, `SHA-1`, `MD5` algorithms. Must be set to `PREFIX` or `POSTFIX`                 | Only for `Salted SHA-512`, `Salted SHA-256`, `Salted SHA-1`, `Salted MD5` algorithms | N/A                              | N/A                              |
+> Note: Because the plain text password is not specified when a hashed password is provided, password policy is not applied.
```
[the remark this answers is not in the document]
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: Made the change.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 1 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: Don't want to imply that should  do an action of setting status to staged, so have edited to: "When updating a user with a hashed password the user must be in the `STAGED` status."

**Pull request merged**